### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by LappleApple**
Thanks for submitting this to FMRM. Good job so far. A few additional things that would be great:

- How are we using it in production, and what evidence of its value can we provide? For example, any info about time saved, work done faster, obstacles removed, etc. will help devs see how this can help them.
- Is anything out there similar? If yes, a line about how this is different will put the project in context. If not, then say that you created it because nothing like it existed already.
- Is the "How to build on your own" section complete? Looks a little skinny.
- Feel free to take a look at this maturity model (https://github.com/LappleApple/feedmereadmes/blob/master/README-maturity-model.md) to get additional ideas about types of content you think will be essential for potential users to get started. It's important to maintain that balance between "just enough documentation" and "too much documentation." Minimal and essential is a good aim. :)

### CODE REVIEW

**Comment by LappleApple** [comment]
👍

File: README.md [hunk 1]
```diff
@@ -4,31 +4,32 @@
...
-By using this library, the state of your [Circuit Breakers](http://martinfowler.com/bliki/CircuitBreaker.html) (closed, open, half-open) will be exposed to your Spring Actuator endpoint.
-
-Just make use of the `@FailsafeBreaker` annotation, to inject a Circuit Breaker where you need it. The state information of the Circuit Breaker will be exposed out of the box and without any additional efforts.
-
-
-The library will be enhanced with furhter features, a description of them can be found in the issue tracker. Still it's already used in production environments by Zalando.
+**Failsafe Actuator** is a Java library that provides a simple monitoring interface for [Spring Boot](https://projects.spring.io/spring-boot/) applications that use the [Failsafe](https://github.com/jhalterman/failsafe) library. Using Failsafe Actuator will readily expose the state of your [circuit breakers](http://martinfowler.com/bliki/CircuitBreaker.html) (closed, open, half-open) to your Spring Actuator endpoint without additional effort. Use the `@FailsafeBreaker` annotation to inject a circuit breaker whereever you need one.
```
Comment (MALPI) [inline README.md]: Should be [Circuit Breakers]

Comment (MALPI) [inline README.md]: Typo: wherever

File: README.md [hunk 2]
```diff
@@ -4,31 +4,32 @@
...
 
 ## Core Technical Concepts/Inspiration
 
-By enabling the support of Springs dependency injection, the usage of *Failsafe* is even easier. 
-
-Having the possibility to monitor the state of a Circuit Breakers is essential for applications which are used in production, since you can directly see if one of the third parties your app is relying on, currently is unavailable. This enables you and your teams to act accordingly and take the right decisions quick.
-
-## Roadmap
+Failsafe Actuator supports Spring's dependency injection to make it easier to use *Failsafe*. It allows you to monitor the state of your Circuit Breakers so that, whenever a third party that your app relies upon suddently becomes unavailable, you can discover it immediately and take action. This is essential for applications used in production.
```
Comment (MALPI) [inline README.md]: Typo: suddenly

**Comment by MALPI** [comment]
:+1:

**Pull request merged**